Rakefile

```diff
@@ -28,4 +28,4 @@ task :clean do
   FileUtils.rm_r('spec/fixtures/derivatives') if File.exist?('spec/fixtures/derivatives')
 end
 
-task :default => [:spec, :rubocop]
+task :default => %i[spec rubocop]
```

Comment:
i am not a fan of ruby "%xxxx" syntax -- looking up each % thingy to figure out what it means, etc -- I think it obscures rather than clarifies.  I'm disinclined to accept this.  DLSS has new hires that aren't rubyists, and it may not be a ruby shop forever.  Code with the "precious rubyisms" as i call them will be harder to understand.

Reply:
The motivation for this rule is that it forces every element in the array to have the same type, so a stray colon or missing quotes cannot cause later operations to break.  I agree in this specific case it might be overkill for 2 elements, not a great benefit.  I chose to enforce the rule, not to edit this line.

This codebase already uses the similar `%w` operator in 6 different places, so I can't really see the objection overall.